doc_user/UserManual.adoc

```diff
@@ -828,6 +828,348 @@ image:image251.png[success]
+* *with movement only*: if only the pharmaceuticals for the selected Medical Type which have been used at least once are needed.
+
+Then click on *[.underline]##O##k* button to insert all the pharmaceuticals that match the choice as the inventory(ies) row(s) in the *Inventory row table*.
+
+===== 4.4.2.2 Edit Inventory ([.underline]##E##dit Inventory)
+
+To modify an inventory, select the inventory in the *_Inventory Browser_* window. Once the inventory has been highlighted, click on *[.underline]##E##dit* button to enter on the *_Edit Inventory_* window. Now the record is available for changes. This function allows editing all the data of the inventory including inventory(ies) row(s) of the inventory.
+
+NOTE: only inventories in DRAFT and VALIDATED states can be edited.
+
+image:image256.png[Edit Inventory]
+
+During modification it is possible to validate the inventory by clicking on the *[.underline]##V##alidate* button. This may results in the following dialog:
+
+image:image271.png[Error]
+
+If the inventory has not been saved yet since last changes.
+
+image:image266.png[Validate Inventory]
+
+This Information window asks wheter to check for any medicals on the stock or just the ones that are in the list?
+
+- *Yes, all medicals*: the system will check if new movements are made on the main store for any medical and eventually show the changes in a new dialog. See below.
+- *No, only the medicals in the list*: the system will check if new movement are made on the main store for the specific medicals that are in the list. E.g. see the image below. 
+
+image:image272.png[Validate]
+
+Click on the *[.underline]##O##k* button to continue. The system may also show another information message that inform the date of the inventory will be updated as a new more recent movement has been found:
+
+image:image273.png[Inventory date update]
+
+NOTE: Since the inventory will result in a number of charges/discharges in <<pharmaceutical-stock,Pharmaceutical Stock>> to align the stock levels to the real quantities (physical count), its date cannot be prior an already existing movement, so the system will always check if the current inventory is the latest operation to be recorded.
+
+Click on the *[.underline]##O##k* button to continue. If new movements have been found, the system will show the following dialog in order to update the current inventory:
+
+image:image274.png[Question]
+
+By clicking the *[.underline]##Y##es* button the system will actualize the inventory by adding new inventory row(s) and by updating also its date/time and its status, see the picture below:
+
+image:image275.png[validate Inventory]
+
+Otherwise, by clicking the *[.underline]##N##o* button, nothing will be changed, but the inventory will remain (or go back) to DRAFT state.
+
+During modification, and only once fully validated, it is possible to Confirm the inventory by clicking on *[.underline]##C##onfirm* button. This will results in the following dialog:
```

Comment:
```suggestion
During modification, and only once fully validated, it is possible to Confirm the inventory by clicking on *[.underline]##C##onfirm* button. This will result in the following dialog:
```